### DISCUSSION TITLE
iris neoforge for 1.20.1

### DISCUSSION THREAD

**Discussion by hgeri06** [IrisShaders/Iris#2565] [Ideas]
i wanna play with some mods in 1.20.1 but iris doesn't have neoforge for 1.20.1 so if possible then make one for this and maybe older versions :)

**Comment by macaron621** [comment]
> i wanna play with some mods in 1.20.1 but iris doesn't have neoforge for 1.20.1 so if possible then make one for this and maybe older versions :)
Oculus
